Makefile

```diff
@@ -1,49 +1,44 @@
-deptools=deptools
 sharness = sharness/lib/sharness
-gx=$(deptools)/gx
-gx-go=$(deptools)/gx-go
 
 # For debugging
 problematic_test = TestClustersReplicationRealloc
 
-export PATH := $(deptools):$(PATH)
-
 all: build
 clean: rwundo clean_sharness
 	$(MAKE) -C cmd/ipfs-cluster-service clean
 	$(MAKE) -C cmd/ipfs-cluster-ctl clean
 	@rm -rf ./test/testingData
 	@rm -rf ./compose
 
-install: gx-deps
+install:
 	$(MAKE) -C cmd/ipfs-cluster-service install
 	$(MAKE) -C cmd/ipfs-cluster-ctl install
 
-docker_install: docker_gx-deps
+docker_install:
```

Comment:
the `docker` target is missing

Reply:
`docker_install` was install but gx would use local deps . Now with `gx`gone, `docker_install` is same as `install`, so probably we can get rid off this target.